admin/client/dist/js/AddToCampaignForm.js

```diff
@@ -22,7 +22,7 @@
   }
 
   _jQuery2.default.entwine('ss', function ($) {
-    $('.add-to-campaign-action, #add-to-campaign__action').entwine({
+    $('#add-to-campaign__dialog .add-to-campaign-action,' + '.cms-content-actions .add-to-campaign-action,' + '#add-to-campaign__action').entwine({
```

Comment:
Doing JS string concat here in addition to the comma separation mandated by CSS selector syntax seems weird - any particular reason?